### ISSUE TITLE
add api.export to support Meteor 0.6.5+

### ISSUE THREAD

**Issue by dknell** [andreioprisan/twilio-meteor#4]
Please add this to package.js for Meteor 0.6.5 support:

``` javascript
Package.on_use(function(api) {
  if (api.export) api.export('Twilio', 'server');
  api.add_files('twilio_npm.js', 'server');
});
```

**Comment by workingBen** [comment]
+1 for Meteor 0.6.5 support

**Comment by andreioprisan** [comment]
Agreed, will make this update for 0.6.5 support.
In the future, just submit a pull request and I'll approve it.

**Comment by andreioprisan** [comment]
this is now fixed in commit 7f36ec52fc334caf8b740320524df62afce890e1
